### PR TITLE
New harvester - SCENT products

### PR DESCRIPTION
**Pull request by davidcordeiro-deimos**
### Features:
New harvester for SCENT data. This harvester allows to collect 8 new collections (id: name):

- "SCENT_DANUBE_IMAGE"
- "SCENT_DANUBE_VIDEO"
- "SCENT_DANUBE_MOISTURE"
- "SCENT_DANUBE_TEMPERATURE"
- "SCENT_KIFISOS_IMAGE"
- "SCENT_KIFISOS_VIDEO"
- "SCENT_KIFISOS_MOISTURE"
- "SCENT_KIFISOS_TEMPERATURE"

This harvester has an ignore list with the metadata fields that should not be indexed in SOLR.

**It is required one harvester per collection**. The mandatory harvester configuration settings is as follows:
{
"wfs_url": "http://147.102.5.93:8080/geoserver/geomesa/wfs",
"wfs_version": "1.1.0",
"collection": "SCENT_DANUBE_IMAGE"
}
- [X] includes new  features
- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes bugfix

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by georgiana-b** [comment]
@davidcordeiro-deimos Could you please also add the documentation to the README?

**Comment by davidcordeiro-deimos** [comment]
@georgiana-b Since we are starting to use the Wiki, I was going to add this harvester information, after the approval of the harvester. Because otherwise, we would be adding information to the Wiki that is not available in the master branch.

File: ckanext/nextgeossharvest/harvesters/scent.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            last_product_index =+ 1
+        else:
+            last_product_index = 0
+
+        wfs = WFS(url=wfs_url, version=wfs_version)
+
+        wfs.set_collection(typename)
+        sortby=['When']
+
+        result = wfs.make_request(max_dataset, sortby, last_product_index)
```
Comment (georgiana-b) [inline ckanext/nextgeossharvest/harvesters/scent.py:109]: From what I understand, in the second job it should gather from the `last_product_index` further. However when I run it locally, I see the the products that have been harvester in the first run being reassessed:

image

Could you please explain/look into it? It seems inneficient to go through the already harvested products every time unless `update_all` is true.

**Comment by georgiana-b** [comment]
As for the documentation, you're right about the Wiki but I think we should also add it to the README just for consistency.

File: ckanext/nextgeossharvest/harvesters/scent.py (hunk 1 above)
Comment (davidcordeiro-deimos) [reply ckanext/nextgeossharvest/harvesters/scent.py:109]: @georgiana-b The commit was not up-to-date. I had a `not None` statement, which has promptly corrected and tested. 

The proper behaviour of the harvester is to harvest 100 **new** datasets per job.

**Comment by davidcordeiro-deimos** [comment]
@georgiana-b As per Issue #102, I thought the objective was to keep the README.md to a bare minimum.

**Comment by georgiana-b** [comment]
I also need to see the expected behaviour documented at the time reviewing the harvester, especially since not all harvesters behave the same way. You have all the information after discussing with the provider and you need to pass that on to me or whoever will need to use the harvester in the future.
The WIKI is a great way to leave a documentation for the system administrators that will take over. 
   
But I said in [102](https://github.com/NextGeoss/ckanext-nextgeossharvest/issues/102#issuecomment-544523017), the changes to the README are welcomed as well, especially since they us the chance to review the documentation for a harvester together with the code for it.

**Comment by georgiana-b** [comment]
Following latest changes, the harvester is now behaving properly. Aside from lacking documentation it's good to go.

**Comment by joaandrade** [comment]
Thank you @georgiana-b 
Is it possible to include this harvester in the release of this week?

**Pull request merged**